components/eamxx/src/physics/mam/eamxx_mam_dry_deposition_process_interface.hpp

```diff
@@ -43,10 +58,10 @@ class MAMDryDep final : public scream::AtmosphereProcess {
   // --------------------------------------------------------------------------
 
   // The type of subcomponent
-  AtmosphereProcessType type() const { return AtmosphereProcessType::Physics; }
+  AtmosphereProcessType type() const override { return AtmosphereProcessType::Physics; } 
```

Comment:
Minor C++ question: Is `override` needed (or good practice) when we are explicitly adding a function body right here?